LamaObjectIdentifier  object
LamaDescriptorIdentifier descriptor
LamaMapAction action
---
LamaObjectIdentifier[] objects
LamaDescriptorIdentifier[] descriptors
